src/Mainstore/Store.js

```diff
@@ -6,14 +6,14 @@ import LatestArticleReducer from '../Landing/redux/reducers/latestArticlesReduce
 import signUpReducer from '../auth/signup/redux/reducer/signUpReducer';
 import allArticleReducer from '../Articles/redux/reducers/allArticlesReducer';
 import loginReducer from '../auth/Login/redux/reducers/loginReducer';
-
+import SearchArticleReducer from '../Articles/redux/reducers/SearchArticleReducer';
 
 const reducers = combineReducers({
   latest_articles: LatestArticleReducer,
   login: loginReducer,
   signUpUser: signUpReducer,
   allArticleReducer,
-
+  searched_articles: SearchArticleReducer,
```

Comment:
@alexxsanya please change and use the camelcase convention as expected of us for this particular project.

Reply:
@neelxie this has been fixed.